lib/cylc/config.py

```diff
@@ -2100,6 +2101,15 @@ def load_graph(self):
                         if offset > old_offset:
                             self.taskdefs[task_name].xclock_label = label
                 else:
+                    try:
+                        if not callable(get_func(xtrig.func_name, self.fdir)):
+                            raise SuiteConfigError("ERROR, xtrigger function "
+                                                   "not callable: "
+                                                   "%s" % xtrig.func_name)
+                    except ImportError:
```

Comment:
Hmmm, just realized you have `ModuleNotFoundError` on `master`, and the parent class `ImportError` on `7.8.x`. I think we better use the parent `ImportError` on master too @dwsutherland . WDYT?

Reply:
@kinow - Yes, as python3 gave me `ModuleNotFoundError`, but python2 produced `ImportError` and the former is not recognised. I didn't try `ImportError` in python3 (Is it back compat?), and figured both have diverged anyway.. Is this ok?

Reply:
Interesting (another hole in my Python 3 knowledge!).

Reply:
I believe `ImportError` is the parent class of `ModuleNotFoundError`.

>Interesting (another hole in my Python 3 knowledge!).

Me too! I thought at least this part was consistent in both. But catching the parent class should be fine, though better to confirm if that works. Perhaps the unit tests could be used to validate that it works?

Reply:
> I believe `ImportError` is the parent class of `ModuleNotFoundError`.

Well I guess the child doesn't exist in python2  ... Consistency aside, would it have been better to use the parent?

Also does consistency matter between 7.8.x and 8? at some point we'll stop back porting changes? (as I guess it matters for that)

Reply:
Good point. If that works as-is, all good then I think